system-contract-dapp-playground/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "system-contract-dapp-playground",
```

Comment:
Take a look at the package.json file in the root dir to see the format of fields and other fields to fill in